Slider/Assets/Scripts/UI/Artifact/AreaArtifacts/OceanArtifact.cs

```diff
@@ -9,17 +9,24 @@ public class OceanArtifact : UIArtifact
     public OceanArtifactHighlights oceanHighlights;
 
     private bool canRotate = true;
+
+    [SerializeField] private GameObject controllerSupportButtonsHolder;
     
     private new void Awake()
     {
         base.Awake();
+
+        Player.GetInstance().ControlSchemeChanged += OnPlayerControlSchemeChanged;
```

Comment:
when you make the other event `public static` you can just do `Player.OnControlSchemeChanged += method` instead

also, subscribe in OnEnable() and unsubscribe in OnDisable()! otherwise when the player is destroyed (going between scenes), stuff will break bc of null references